src/components/UI/KubernetesVersionLabel.tsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+interface IKubernetesVersionLabelProps {
+  version: string;
+}
+
+const KubernetesVersionLabel = ({ version }: IKubernetesVersionLabelProps) => {
```

Comment:
We usually use this pattern

```suggestion
const KubernetesVersionLabel: React.FC<IKubernetesVersionLabelProps> = ({ version }) => {
```

Reply:
Look, I have no idea what I'm doing there. Just copied and modified `ValueLabel.tsx` 😜